website/docs/r/workloads_sap_single_node_virtual_instance.html.markdown

```diff
@@ -0,0 +1,323 @@
+
+* `os_sap_configuration` - (Required) An `os_sap_configuration` block as defined below. Changing this forces a new resource to be created.
+
+* `sap_product` - (Required) The SAP Product type for the SAP Single Node Virtual Instance. Possible values are `ECC`, `Other` and `S4HANA`. Changing this forces a new resource to be created.
+
+* `single_server_configuration` - (Required) A `single_server_configuration` block as defined below. Changing this forces a new resource to be created.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `managed_resource_group_name` - (Optional) The name of the managed Resource Group for the SAP Single Node Virtual Instance. Changing this forces a new resource to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the SAP Single Node Virtual Instance.
+
+---
+
+An `os_sap_configuration` block supports the following:
+
+* `sap_fqdn` - (Required) The FQDN of the SAP system. Changing this forces a new resource to be created.
+
+* `deployer_virtual_machine_packages` - (Optional) A `deployer_virtual_machine_packages` block as defined below. Changing this forces a new resource to be created.
+
+---
+
+A `deployer_virtual_machine_packages` block supports the following:
+
+* `storage_account_id` - (Required) The ID of the deployer VM packages storage account. Changing this forces a new resource to be created.
+
+* `url` - (Required) The URL of the deployer VM packages file. Changing this forces a new resource to be created.
+
+---
+
+A `single_server_configuration` block supports the following:
+
+* `app_resource_group_name` - (Required) The name of the application Resource Group where SAP system resources will be deployed. Changing this forces a new resource to be created.
+
+~> **Note:** While creating SAP Single Node Virtual Instance, service would provision the extra SAP system/component in `app_resource_group_name` which aren't defined in tf config. At this time, when `app_resource_group_name` is different with the Resource Group where SAP Single Node Virtual Instance exists, we can set `prevent_deletion_if_contains_resources` to `false` to delete all resources defined in tf config and the resources created in `app_resource_group_name` with `tf destroy`. But when `app_resource_group_name` is same with the Resource Group where SAP Single Node Virtual Instance exists, some resources like the subnet defined in tf config cannot be deleted with `tf destroy` since the resources defined in tf config are being referenced by the SAP system/component created in `app_resource_group_name`. So it has to manually delete the resources in `app_resource_group_name` first after the SAP Single Node Virtual Instance is deleted, and then the resources in tf config can be deleted successfully for this situation.
```

Comment:
```suggestion
~> **Note:** While creating an SAP Single Server Virtual Instance, the service would provision the extra SAP system/component in the `app_resource_group_name` that isn't defined in the HCL Configurations. At this time, if the `app_resource_group_name` is different from the Resource Group where SAP Single Server Virtual Instance exists, you can set `prevent_deletion_if_contains_resources` to `false` to delete all resources defined in the HCL Configurations and the resources created in the `app_resource_group_name` with `terraform destroy`. However, if the `app_resource_group_name` is the same with the Resource Group where SAP Single Server Virtual Instance exists, some resources such as the subnet defined in the HCL Configurations cannot be deleted with `terraform destroy` since the resources defined in the HCL Configurations are being referenced by the SAP system/component. In this case, you have to manually delete the SAP system/component before deleting the resources in the HCL Configurations.
```

Reply:
Updated